src/Components/Feed/ImageCarousel.js

```diff
@@ -0,0 +1,65 @@
+import { useState } from "react";
+
+//<ImageCarousel urlArray = {filePreviewArray} />
+export function ImageCarousel(props) {
+    const [currentImageIndex, setCurrentImageIndex] = useState(0)
+
+    const rotateCurrentImage = (direction) => {
```

Comment:
"rotateCurrentImage" made me think we have a feature for users to rotate the orientation of their images - can change in next iteration. 

Why is this up and down? Shouldn't it be left/right?